frontend/src/concepts/pipelines/content/tables/pipelineVersion/PipelineVersionTableRow.tsx

```diff
@@ -78,6 +81,10 @@ const PipelineVersionTableRow: React.FC<PipelineVersionTableRowProps> = ({
                   ),
                 );
               },
+              isAriaDisabled: isCreateDisabled,
+              tooltipProps: isCreateDisabled
+                ? { content: 'Cannot create run for Kubeflow v1 SDK pipelines' }
```

Comment:
same here. If this is a const, we don't have to worry about typos and making the same change in different strings.